_posts/2019/2019-07-28-consider-value-objects.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Consider Value Objects"
+description: "Making text readable on"
+layout: article
+category: ruby
+image:
+  base: '2019/consider-value-objects'
+  alt: 'Cubes'
+  credit: Christian Fregnan
+  source: "https://unsplash.com/photos/TAVKURx-xLw"
+
+---
+
+Often we use simple concepts in our app, that don't start out as objects.
+
+You'll often find yourself creating multiple view helpers or passing a value (or several) around several methods, or within a model.
+
+This loose concept can be refactored into a “Value Object”.
```

Comment:
I think these three opening sentences need re-working. I was confused.

Maybe something like:

- You'll often find yourself doing x, y, z...
- This happens in cases where you want something simpler than a fully-blown objecct
- In these cases, consider refactoring into a 'value object'.

A structure like that. What do you think?